kratos/utilities/geometrical_projection_utilities.h

```diff
@@ -168,6 +168,33 @@ class GeometricalProjectionUtilities
             EchoLevel);
     }
 
+    /**
+     * @brief Project a point over a line (2D or 3D)
+     * @tparam TGeometryType The type of the line
+     * @param rGeom The line where to be projected
+     * @param rPointToProject The point to be projected
+     * @param rPointProjected The point pojected over the line
+     * @return Distance The distance between point and line
+     * source: https://www.qc.edu.hk/math/Advanced%20Level/Point_to_line.htm "Method 3 Using Dot Product"
```

Comment:
@loumalouomega do you know how to properly quote a source in doxygen? I couldn't find it

Reply:
http://www.doxygen.nl/manual/commands.html